rfcs/0000-community-coordination-hub.md

```diff
@@ -0,0 +1,150 @@
+---
+feature: community_coordination_hub
+start-date: 2020-07-30
+author: David Arnold <dar@xoe.solutions>
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Adopt goals, spirit, concept and execution of [`kubernetes/community`]
+where it _does_ fit [us](https://nixos.org/).
+
+[`kubernetes/community`]: https://github.com/kubernetes/community
+
+# Motivation
+[motivation]: #motivation
+
+We lack an optimum level of _empowerment_ on distinct vital aspects for
+the best [leverage](https://nav.al/least) on the available resources:
```

Comment:
@blaggacao I forgot to review this section after [your encouragement](https://github.com/blaggacao/rfcs/pull/1#issuecomment-671574731), sorry :). As a start, this sentence is mostly well understood by me, so I think it's mostly fine, but I don't understand the end of it and the list that follows. I mean, how exactly "Efficient Management Processes" are a resource? I think this small change makes sense and is much better? I hope I got your point right:

```suggestion
We lack an optimum level of _empowerment_ on distinct vital aspects for
the best [leverage](https://nav.al/least). We don't have:
```